sympy/geometry/point.py

```diff
@@ -80,49 +82,225 @@ class Point(GeometryEntity):
+            truncate a nonzero dimension
+        TypeError : an element was not a Point instance
+
+        Examples
+        ========
+
+        >>> from sympy.geometry import Point
+
+        By default, extra dimensions with zero elements are trimmed:
+
+        >>> Point.normalize_dimensions(Point(1, 2, 0), Point(1, 2, 3, 0))
+        [Point3D(1, 2, 0), Point3D(1, 2, 3)]
+
+        Zeros will be padded onto Points that are too short:
+
+        >>> a, b = Point(1, 2), Point(1, 2, 3)
+        >>> Point.normalize_dimensions(a, b)
+        [Point3D(1, 2, 0), Point3D(1, 2, 3)]
+
+        The keyword `dim` will force a given dimension:
+
+        >>> Point.normalize_dimensions(a, b, dim=4)
+        [Point(1, 2, 0, 0), Point(1, 2, 3, 0)]
+
+        But iff truncation would result in losing a non-zero dimension,
+        a ValueError is raised:
+
+        >>> Point.normalize_dimensions(a, b, dim=2)
+        Traceback (most recent call last):
+        ...
+        ValueError: Point has too many dimensions
+        """
+        if not points:
+            raise ValueError('Must provide 1 or more Points.')
+        if not all(isinstance(i, Point) for i in points):
+            raise TypeError('all elements must be Points')
+        minimal = not kwargs.get('max', False)
+        dim = kwargs.get('dim',
+            max(i._size for i in points) if minimal else
+            max(i.ambient_dimension for i in points))
+        if not all(a.ambient_dimension == dim for a in points):
+            points = [Point(a, dim=dim) for a in points]
+        return points
+
     def __contains__(self, item):
         return item in self.args
 
-    def is_concyclic(*args):
-        # Coincident points are irrelevant and can confuse this algorithm.
-        # Use only unique points.
-        args = list(set(args))
-        if not all(isinstance(p, Point) for p in args):
-            raise TypeError('Must pass only Point objects')
+    @staticmethod
+    def are_coplanar(*points):
+        """Return True if the points lie in a unique plane.
+
+        All two dimensional points are coplanar: they lie in the
+        x-y plane.  Three dimensional points are coplanar only
+        if they are not collinear and thus define a unique plane.
+
+        Parameters
+        ==========
+
+        A set of points
+
+        Raises
+        ======
+
+        ValueError : if there is a dimension mismatch in point given.
+        NotImplementedError : for dimensions higher than 3
+
+        Returns
+        =======
+
+        boolean
+
+        Examples
+        ========
 
+        >>> from sympy import Point3D
+        >>> p1 = Point3D(1, 2, 2)
+        >>> p2 = Point3D(2, 7, 2)
+        >>> p3 = Point3D(0, 0, 2)
+        >>> p4 = Point3D(1, 1, 2)
+        >>> Point3D.are_coplanar(p1, p2, p3, p4)
+        True
+        >>> p5 = Point3D(0, 1, 3)
+        >>> Point3D.are_coplanar(p1, p2, p3, p5)
+        False
+
+        """
+        if not points:
+            return False
```

Comment:
In order to answer the query of whether something lies in the same plane, one must have a plane to talk about...which doesn't exist with less than 3 pts. What about a ValueError in such cases?